atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -44,6 +44,28 @@
 
     @Value.Default
     public int poolSize() {
+        return 30;
```

Comment:
Yeah true, that could be a lot for large clusters. Do you see any benefit to having the max size be a multiple of the min? Like if I change it to explicit defaults of 30 and 100 do you think that would screw anyone up that has non-default poolSize's configured?